model/operation.cc

```diff
@@ -618,48 +573,48 @@ namespace
     cairo_stroke_preserve(cairo);    
     
     if (coupled())
-     {
-       float ivo=intVarOffset*z;
-       cairo_new_path(cairo);
-       cairo_move_to(cairo,r,0);
-       cairo_line_to(cairo,r+ivo,0);
-       cairo_set_source_rgb(cairo,0,0,0);
-       cairo_stroke(cairo);
+      {
+        float ivo=intVarOffset*z;
+        cairo_new_path(cairo);
+        cairo_move_to(cairo,r,0);
+        cairo_line_to(cairo,r+ivo,0);
+        cairo_set_source_rgb(cairo,0,0,0);
+        cairo_stroke(cairo);
      
-       // display an integration variable next to it
-       RenderVariable rv(*intVar, cairo);
-       // save the render width for later use in setting the clip
-       intVarWidth=rv.width()*z;
-       if (rv.width()<intVar->iWidth()) intVarWidth=intVar->iWidth()*z;
-       // set the port location...
-       intVar->moveTo(x()+r+ivo+intVarWidth, y());
+        // display an integration variable next to it
+        RenderVariable rv(*intVar, cairo);
+        // save the render width for later use in setting the clip
+        intVarWidth=rv.width()*z;
+        if (rv.width()<intVar->iWidth()) intVarWidth=0.5*intVar->iWidth()*z;
```

Comment:
Shouldn't the condition be <0.5*intVar->iWidth()*z? Also, this seems to be a ratcheting mechanism, ie scales one way, but not the other...

Reply:
I'm going to merge this anyway, but bear in mind I suspect this is not quite right.

Reply:
Sorry, I was working on this, and had changed those conditions to be scalable in both directions. But to make the correction on bugfix-1204, I discarded all the changes on this branch (a few tests were failing and I hadn't quite got the bottom of the problem yet).

Reply:
I'll try again tomorrow morning. Sorry, I don't know where the problem with the test slipped in and was still diagnosing it when I switched to bugfix-1204...

Reply:
Misunderstood the question about the 0.5 factor because I didn't see it... fixing that now.